test/FastTests/Server/Documents/Indexing/MapReduce/GroupByComplexObjects.cs

```diff
@@ -205,5 +125,85 @@ public class GroupByComplexObjects : RavenLowLevelTestBase
                 }
             }
         }
+
+        private static void put_docs(DocumentsOperationContext context, DocumentDatabase database)
```

Comment:
Just a convention that private methods in test classes (something like private helpers) are lower cased. I will change that.